language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -607,14 +607,14 @@ buildPackages sdkVersion optScope optOutputDir dependencies = do
     build :: String -> IO ()
     build pkg = do
       putStrLn $ "Building " <> pkg
-      callProcessSilent "yarn" ["run", "tsc", "--project", pkg </> "tsconfig.json"]
+      yarn ["run", "tsc", "--project", pkg </> "tsconfig.json"]
       copyDirectory pkg $ "node_modules" </> scope </> pkg
 
-    callProcessSilent :: FilePath -> [String] -> IO ()
-    callProcessSilent cmd args = do
-      (exitCode, _, err) <- readProcessWithExitCode cmd args ""
+    yarn :: [String] -> IO ()
```

Comment:
Why are you not using `callProcessSilent` here?

Reply:
Didn't feel right having this executable depend on a test utility library

Reply:
Maybe `callProcessSilent` (etc) should be in da-hs-base.

Reply:
Not sufficiently motivated to do it for this one case today but my arm can be twisted if you feel strongly about it :) Do you?

Reply:
@associahedron

Reply:
Not strongly, no :)

Reply:
I do agree though, just inclined to do it down the road when a more motivating circumstance arises.